integration-cli/docker_api_inspect_test.go

```diff
@@ -31,24 +32,22 @@ func (s *DockerSuite) TestInspectApiContainerResponse(c *check.C) {
 		endpoint := fmt.Sprintf("/v%s/containers/%s/json", cs.version, cleanedContainerID)
 
 		status, body, err := sockRequest("GET", endpoint, nil)
-		c.Assert(status, check.Equals, http.StatusOK)
-		c.Assert(err, check.IsNil)
+		c.Assert(status, checker.Equals, http.StatusOK)
+		c.Assert(err, checker.IsNil)
 
 		var inspectJSON map[string]interface{}
-		if err = json.Unmarshal(body, &inspectJSON); err != nil {
-			c.Fatalf("unable to unmarshal body for version %s: %v", cs.version, err)
-		}
+		c.Assert(json.Unmarshal(body, &imageJSON), check.IsNil, Commentf("unable to unmarshal body for version %s: %v", cs.version, json.Unmarshal(body, &imageJSON))
```

Comment:
This ain't gonna work:
- It's `inspectJSON` instead of `imageJSON`
- you call twice `json.Unmarshal` but I don't even think you need a comment here.